cmd/janus/init.go

```diff
@@ -58,6 +63,26 @@ func initDistributedTracing() io.Closer {
 	return closer
 }
 
+func initCircuitBreaker() {
+	// Setup metrics for circuit breaker
+	c, err := plugins.InitializeStatsdCollector(&plugins.StatsdCollectorConfig{
+		StatsdAddr: globalConfig.Stats.DSN,
+		Prefix:     globalConfig.Stats.Prefix,
```

Comment:
Current `stats-go` version used in Janus is 0.4.3, but as of 0.5.0 prefix became part of the DSN. So maybe it makes sense to update stats library in this PR. Moreover - even in 0.4.3 DSN can be different from `statsd` - see https://github.com/hellofresh/stats-go#key-features

Reply:
:+1: